custodia/secrets.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2015  Custodia Project Contributors - see LICENSE file
+# (C) 2015  Custodia Project Contributors - see LICENSE file
```

Comment:
And I'll change for my name :P Sorry about that, I'll send a new commit removing this unrelated change.